statchatui/src/Compare/Compare.jsx

```diff
@@ -21,10 +21,53 @@ const Compare = ({ signedInUserData }) => {
   const [isLoading, setIsLoading] = useState(false);
   const [otherUserStats, setOtherUserStats] = useState(null);
   const [signedInUserStats, setSignedInUserStats] = useState(null);
+  const [userQueue, setUserQueue] = useState([]);
+  const [showStats, setShowStats] = useState(null);
```

Comment:
This is a bit confusing since `showStats` sounds like it would be a boolean. I'd name it something like `currentStatsIndex` or `shownStatsIndex` to make it a bit clearer what it is.